ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -41,7 +41,21 @@ Feature: Notification creation
         And I enter 1 into 'FormattedDiagnosisDate_Month'

```

Comment:
Are you on a branch before my autocomplete changes? As that broke somethign (because autocomplete is not a normal select input) - will fix as part of my ticket

Reply:
umm, most likely - yes :P 
is there anything for me to do here?

Reply:
No I'll fix them, have already started looking into how!